event/datadeal/data_verification_event.go

```diff
@@ -53,61 +45,61 @@ func (d DataVerificationEvent) EventHandler(event ctypes.ResultEvent) error {
 		return err
 	}
 
-	voteOption, err := d.verifyAndGetVoteOption(dealID, dataHash)
+	voteOption, err := e.verifyAndGetVoteOption(dealID, dataHash)
 	if err != nil {
-		log.Infof("can't vote due to error while verify. dealID(%d). dataHash(%s)", dealID, dataHash)
+		log.Infof("vote No due to error while verify. dealID(%d). dataHash(%s)", dealID, dataHash)
 	}
 
 	msgVoteDataVerification, err := makeDataVerificationVote(
-		d.reactor.OracleAcc().GetAddress(),
+		e.reactor.OracleAcc().GetAddress(),
 		dataHash,
 		dealID,
 		voteOption,
-		d.reactor.OraclePrivKey().Serialize(),
+		e.reactor.OraclePrivKey().Serialize(),
 	)
 	if err != nil {
 		return err
 	}
 
-	txBuilder := panacea.NewTxBuilder(*d.reactor.QueryClient())
+	txBuilder := panacea.NewTxBuilder(*e.reactor.QueryClient())
 
-	txBytes, err := generateTxBytes(msgVoteDataVerification, d.reactor.OracleAcc().GetPrivKey(), d.reactor.Config(), txBuilder)
+	txBytes, err := generateTxBytes(msgVoteDataVerification, e.reactor.OracleAcc().GetPrivKey(), e.reactor.Config(), txBuilder)
```

Comment:
If merged my [PR](https://github.com/medibloc/panacea-doracle/pull/91/files#diff-1800206db049e385790536e9cc7a7c5eb8bc74dfaef4039c144c45dc5d0418abR25), you can used this function `txBuilder.GenerateTxBytes(...)

Reply:
Good! Reflected in [dc45af6](https://github.com/medibloc/panacea-doracle/pull/95/commits/dc45af634288b9de9e1ce6b42d75b77bbeaf3625). Can we refactor our `broadcast` function in this same way? wdyt?

Reply:
Good, But...
This function using grpcClient and simply priting result log.
So, the 'broadcast' function seems to vary depending on the each event.

Reply:
I thought that different error messages for each event can be created outside the `broadcast` function, and `broadcast` just check `TxResponse.Code` and return height & hash if the transaction succeed. How about this?

Reply:
Therefore, Where would you like to create `broadcast` function?

Reply:
And when a transaction fails, each event can have different error handling.
If only returned height and hash, we cannot handling error

Reply:
For now, we don't handle any error when tx fails, so I think it could be good.
If we make `broadcast` more general, I would make a function of `service` struct :)

Reply:
OK.
If so, please refactorying this PR according to your opinion. @audtlr24

Reply:
After the PR #91 is applied, I will apply it at once :)